library/alloc/src/vec/mod.rs

```diff
@@ -2568,6 +2568,8 @@ impl<T, A: Allocator> Vec<T, A> {
             }
             unsafe {
                 ptr::write(self.as_mut_ptr().add(len), element);
+                // since next() executes user code which can panic we have to bump the length
+                // after each read
```

Comment:
What is "read" referring to? I see a "write" above, and of course the "next".
```suggestion
                // Since next() executes user code which can panic we have to bump the length
                // after each read.
```

Reply:
I'll rephrase it to "each step" instead since it's more general.